app/controllers/api/v1/neighbors_controller.rb

```diff
@@ -0,0 +1,35 @@
+class Api::V1::NeighborsController < Api::ApiController
+  include ControllerHelper
+  before_action :neighbor, only: :update
+
+  def index
+    @neighbors = current_user.neighbors
+  end
+
+  def update
+    if @neighbor.update(neighbor_params)
+      @neighbor.reindex
+      render json: @neighbor
+    else
+      render json: @neighbor.errors, status: :unprocessable_entity
+    end
+  end
+
+  def search
+    @neighbors = User.search(params[:approved],
```

Comment:
as approved is a boolean
would look better in where with osbb_id and id
search by some names phone number whould be better

Reply:
refactor